### ISSUE TITLE
Add missing logic to pathogen tests for samples assigned to contacts

### ISSUE THREAD

**Issue by MateStrysewskeSym** [hzi-braunschweig/SORMAS-Project#2143]
### Situation Description
Since #1753, samples can now also be assigned to contacts. There is some logic associated with pathogen tests that only makes sense for samples assigned to cases, and this logic will be removed for samples exclusively assigned to contacts. After #1753 is done, this logic should be re-added and re-written.

### Feature Description
- [x] When the contact is unconfirmed and not converted to a case, when saving a positive and verified pathogen test WITH THE SAME DISEASE as the contact, a popup should open and ask the user whether they want to confirm the contact and convert it to a case; if the user agrees, the contact classification should be changed to confirmed and the case creation window should be opened just like when the user would've manually clicked the "Convert this contact to a case" button.
- [x] When the contact is unconfirmed and not converted to a case, when saving a positive and verified pathogen test WITH A DIFFERENT DISEASE than the contact, a popup should open and ask the user whether they want to create a case for the contact person with the new disease. If the user agrees, the case creation window should open with the disease of the pathogen test already selected and the personal details of the contact filled in.

**Comment by bernardsilenou** [comment]
@MateStrysewskeSym 
We need to activate the same logic event participant  or screening participants. See #2340 , 5,4 and 5,5

**Comment by FredrikSchaeferVitagroup** [comment]
This comment is just for documentation purposes. No reaction is necessary.

There was some decision to be made with the second requirement:

When the contact is unconfirmed and not converted to a case, when saving a positive and verified pathogen test WITH A DIFFERENT DISEASE than the contact,
a popup should open and ask the user whether they want to create a case for the contact person with the new disease.
If the user agrees, the case creation window should open with the disease of the pathogen test already selected and the personal details of the contact filled in.

### Current situation

There is no already defined functionality to do that. You can either create a case from scratch, or you can create a case from a contact.
In the latter the contact disease is automatically transfered to the case. The requirements demands to set another disease, though.

Methods calls are as follows:

CaseController.createFromContact(...)
| |
\\/
 CaseController.getCaseCreateComponent(...)

> separates procedure for contact and eventPartcipant

| |
\\/
CaseDataDto.buildFromContact(...)

> automatically chooses contact.Disease and transfers symptoms afterwards

| |
\\/
CaseDataDto.build(Person, Disease)


### Solution ideas:

copy and adapt the first three methods, transfer duplicate code to private methods
+ procedure clearly separated for creation of cases with same and different disease as contact
- relatively many new methods

use existing methods, adapt case attributes afterwards
- bad procedure

Implement disease as additional parameter in CaseController.getCaseCreateComponent(...)
+ (seemingly) fits to the purpose of the method
+ not much duplicate code
- CaseController.getCaseCreateComponent(...) gets slightly more complex
- probably new method CaseDataDto.buildFromUnrelatedContact(...) necessary

### Implementation:

3rd solution chosen.

**Comment by FredrikSchaeferVitagroup** [comment]
The web part is implemented in this issue. The app part can only be implmented after #2133 is done. I will create a new issue for that.

**Comment by FredrikSchaeferVitagroup** [comment]
Automatically setting contactClassification to CONFIRMED does not work (anymore).
